### PR TITLE
CLDR-14194 Add spec for units structure

### PR DESCRIPTION
**Pull request by macchiati**
##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14194
- [ ] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Comment by macchiati** [comment]
I will need to do more work on units, but wanted to get this first tranche in so that people could see the HTML versions and comment on them.

File: docs/ldml/tr35-general.html [hunk 1]
```diff
@@ -2462,18 +2485,42 @@ <h3>6.4 <a name="perUnitPatterns" href="#perUnitPatterns" id=
...
     denominator with another unit name. For example, a form such as
     "{0} per second" can be used to form "2 feet <strong>per
     second</strong>".</p>
     <p>The difference between these is that in some inflected
     languages, the compoundUnit cannot be used to form grammatical
     phrases. This is typically because the "per" + "second" combine
-    in a non-trivial way. For such languages, the compoundUnit
+    in a non-trivial way. </p>
+    <p>&nbsp;</p>
+    <p>For a description of how to use these fields to construct a localized name, see </p>
```
Comment (pedberg-icu) [inline docs/ldml/tr35-general.html:2518]: see what? The link seems to be missing here

**Review by pedberg-icu — CHANGES_REQUESTED** [review]
Need to fix one missing link. Also please rebase on master (to resolve merge conflicts) and squash (to one commit).

**Comment by macchiati** [comment]
Fixed the dangling sentence, resolved, and squashed.

File: docs/ldml/tr35-general.html (hunk 1 above)
Comment (macchiati) [reply docs/ldml/tr35-general.html]: fixed

**Review by pedberg-icu — APPROVED** [review]

File: docs/ldml/tr35-general.html [hunk 2]
```diff
@@ -1361,9 +1368,27 @@ <h2>6 <a name="Unit_Elements" href="#Unit_Elements" id=
 "color: blue">{0} day</span>&lt;/unitName&gt;
   &lt;unitPattern count="other"&gt;<span style=
 "color: blue">{0} days</span>&lt;/unitName&gt;
+&lt;/unit&gt;</pre>
+    <p class='changed'>The german rules are more complicated, because German has both gender and case. They thus have additional information, as illustrated below. Note that if there is no @case attribute, for backwards compatibility the implied case is nominative. The possible values for @case are listed in the <strong>grammaticalFeatures</strong> element. These follow the inheritance specified in Part 1, Section <a href=
```
Comment (hugovdm) [inline docs/ldml/tr35-general.html:1372]: If I go over this in fine detail and notice nits like "german" vs "German", what's the best place to collect feedback and suggestions now?

Comment (hugovdm) [reply docs/ldml/tr35-general.html:1372]: (I see it's live at https://unicode-org.github.io/cldr/ldml/tr35-general.html)

**Comment by macchiati** [comment]
Right. Further work needs to be done, but it is easier to review the HTML
doc, and collect additional comments for an additional PR. Best to add
comments to the ticket at
https://unicode-org.atlassian.net/browse/CLDR-14194

Mark

**Pull request merged**